mobile/ios/ios-emulator-services.ts

```diff
@@ -37,7 +39,43 @@ class IosEmulatorServices implements Mobile.IEmulatorPlatformServices {
 			this.startEmulatorCore(app, emulatorOptions);
 
 		}).future<void>()();
-	}
+    }
+
+    startAppWithDebugger(app: string, emulatorOptions?: Mobile.IEmulatorOptions) {
+        var tnsIosPackage = this.$npm.install("tns-ios", ["g"]).wait();
```

Comment:
Is it possible to require sudo permissions here?
If node is installed with sudo, -g option may need sudo (or I'm totally incorrect).